ansible-playbook.yml

```diff
@@ -0,0 +1,74 @@
+---
+- name: Install and run todo app on VMs
+  hosts: exercise-4-group
+  remote_user: ec2-user
+
+  vars_prompt:
+  - name: trello_api_key
+    prompt: Enter trello api key
+    private: no
```

Comment:
I mean, this should be private, but the only functional change is it hides it when typing in your terminal so not a big deal.

Reply:
Yeah I figured for debugging it was easier to see that I'd copied things over correctly - will set to private